### PR TITLE
feat: add Keptn support

### PR DESCRIPTION
**Pull request by thisthat**
# Changes

- swap URLs for liveness and readiness
- add recommended K8s labels
- add Makefile to install Keptn, Grafana, Jaeger, and Prometheus
- add a Keptn Post-deployment task to print some contextual information about the service being correctly deployed 

The logs of the post-task can be fetched with `kubectl get pods | grep post-stdout-notification`, which prints the pod name.
The pod name can be used to run `kubectl logs <pod-name>`
Example of output: 
```
{"workloadName":"agenda-service-agenda-service","appName":"agenda-service","appVersion":"","workloadVersion":"v1.0.0","taskType":"post","objectType":"Workload"}
```
# Note
This assumes that the release namespace used by Helm is annotated with `keptn.sh/lifecycle-toolkit: "enabled"`.
If the default namespace is used, please run `kubectl annotate ns default keptn.sh/lifecycle-toolkit: "enabled"`

### CODE REVIEW

File: chapter-9/support/observability/Makefile [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+	#kubectl delete -f config/prometheus/ --ignore-not-found=true
+	#kubectl delete -f config/prometheus/setup --ignore-not-found=true
+	kubectl delete ns observability --ignore-not-found=true
+	kubectl delete ns monitoring --ignore-not-found=true
+	@echo ""
+	@echo "##########################"
+	@echo "Observability Demo deleted"
+	@echo "##########################"
+
+.PHONY: deploy-podtatohead
```
Comment (salaboy) [inline chapter-9/support/observability/Makefile]: @thisthat we can remove all this right?

File: chapter-9/support/observability/Makefile [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+	@echo ""
+	@echo "#######################################################"
+	@echo "Observability Demo installed"
+	@echo "- Install PodTatoHead: make deploy-podtatohead"
+	@echo "- Port-Forward Grafana: make port-forward-grafana"
+	@echo "#######################################################"
+
+.PHONY: install-cert-manager
+install-cert-manager:
```
Comment (salaboy) [inline chapter-9/support/observability/Makefile:54]: @thisthat do we need cert-manager?

Comment (thisthat) [reply chapter-9/support/observability/Makefile:54]: yes, this is needed for Jaeger

File: chapter-9/support/observability/Makefile (hunk 1 above)
Comment (thisthat) [reply chapter-9/support/observability/Makefile]: Yes! Good catch! Removed :)

**Pull request merged**